packages/antd/src/templates/FieldHelpTemplate/index.tsx

```diff
@@ -0,0 +1,27 @@
+import { helpId, FieldHelpProps, FormContextType, RJSFSchema, StrictRJSFSchema } from '@rjsf/utils';
+import { RichHelp } from '@rjsf/core';
+
+/** The `FieldHelpTemplate` component renders any help desired for a field
+ *
+ * @param props - The `FieldHelpProps` to be rendered
+ */
+export default function FieldHelpTemplate<
+  T = any,
+  S extends StrictRJSFSchema = RJSFSchema,
+  F extends FormContextType = any,
+>(props: FieldHelpProps<T, S, F>) {
+  const { fieldPathId, help, uiSchema, registry } = props;
+  if (!help) {
+    return null;
+  }
+
+  const id = helpId(fieldPathId);
+
+  return (
+    <div id={id} className='ant-form-item-extra'>
+      <div className='help-block'>
+        {typeof help === 'string' ? help : <RichHelp help={help} registry={registry} uiSchema={uiSchema} />}
+      </div>
+    </div>
+  );
```

Comment:
Is the extra div and `ant-form-item-extra` required? And also, doesn't the `RichHelp` already handle the `help` is string case?
```suggestion
  return (
    <div id={helpId(fieldPathId)} className='help-block'>
      <RichHelp help={help} registry={registry} uiSchema={uiSchema} />
    </div>
  );
```